providers/oidc.go

```diff
@@ -83,6 +85,7 @@ func (p *OIDCProvider) Redeem(ctx context.Context, redirectURL, code, codeVerifi
 		},
 		RedirectURL: redirectURL,
 	}
+	ctx = oidc.ClientContext(ctx, requests.DefaultHttpClient)
```

Comment:
This seems like a lot of duplication. Would it be possible to configure it globally for all outgoing requests?

Reply:
Maybe have a look at
https://github.com/go-swagger/go-swagger/issues/1563#issuecomment-393223595

Reply:
Yes, that is a more sophisticated way, I can change it to.

> This seems like a lot of duplication. Would it be possible to configure it globally for all outgoing requests?

No, it would require changes to the oauth2 and oidc library

Reply:
Changed in b6da03604981fec57adbaf37e7e7d16e6cddd6a8